pkg/specsyncer/spec2db/controller/placementbinding_spec_sync.go

```diff
@@ -6,16 +6,28 @@ package controller
 import (
 	"fmt"
 
+	"github.com/stolostron/hub-of-hubs-manager/pkg/constants"
 	"github.com/stolostron/hub-of-hubs-manager/pkg/specsyncer/db2transport/db"
 	"k8s.io/apimachinery/pkg/api/equality"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	policyv1 "open-cluster-management.io/governance-policy-propagator/api/v1"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
 )
 
 func AddPlacementBindingController(mgr ctrl.Manager, specDB db.SpecDB) error {
+	placementBindingPredicate, _ := predicate.LabelSelectorPredicate(metav1.LabelSelector{
```

Comment:
I found that the finalizer is added for the local-resource. that is why add back.